docs/requirements.md

```diff
@@ -2,6 +2,12 @@
 
 This document explains the requirements and recommendations in the underlying databases of ScalarDB to make ScalarDB applications work correctly.
 
+## Common requirements
+
```

Comment:
I'd like to add a sentence after this heading so that the user has some understanding for what this section is about.

The following sentence is pretty basic, but please feel free to add more context if the reader should know more🙇🏻‍♂️

```suggestion
## Common requirements

This section lists common requirements when using ScalarDB.

```

Reply:
Thanks for the suggestion. Just a confirmation. Is it OK to say "list" even if we only have a single item (regarding the privileges)?

Reply:
True. Let's use a more generic phrase, like `describes` (or `contains` or `includes`) instead.

```suggestion
## Common requirements

This section describes common requirements when using ScalarDB.

```

Reply:
Thank you! Fixed in 4412756.